SwiftWisdomTests/NSDataTrimTests.swift

```diff
@@ -11,29 +11,15 @@ import XCTest
 
 
 class NSDataTrimTests: XCTestCase {
-    let ProductName = "Hello, Headphone Name!".data(using: String.Encoding.utf8)!
-    let ProductInfoProductNameRawPacketPrefix = "<0102a616>"
-        .ip_dataFromHexadecimalString()!
-    
-    let extraAppendedData = "2342 afa2 ffed 2222"
-        .ip_dataFromHexadecimalString()!
-    
-    lazy var ProductInfoProductNameRawPacketData: Data = {
-        let mutable = NSMutableData()
-        mutable.append(self.ProductInfoProductNameRawPacketPrefix)
-        mutable.append(self.ProductName)
-        mutable.append(self.extraAppendedData)
-        return (NSData(data: mutable as Data) as Data)
-    }()
     
     func testTrimRange() {
         let data = "aa 11 11 11 11 11 bb".ip_dataFromHexadecimalString()!
-        let mutable = NSData(data: data) as Data
+        let mutable = NSMutableData(data: data as Data)
         mutable.ip_trimRange(0...3)
-        XCTAssert(mutable == data.ip_suffixFrom(4)!)
+        XCTAssert(mutable as NSData == data.ip_suffixFrom(4))
```

Comment:
i just noticed this, but we rely on other of our methods for this test. if possible, lets try to isolate this. otherwise, lets leave a TODO here and below.